src/autogluon/bench/frameworks/multimodal/exec.py

```diff
@@ -0,0 +1,103 @@
+import argparse
+import json
+import os
+from datetime import datetime
+from typing import List
+
+from autogluon.core.utils.loaders import load_pd
+from autogluon.multimodal import MultiModalPredictor
+from sklearn.model_selection import train_test_split
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--data_path",
+        type=str,
+        help="Can be one of: dataset name, local path, S3 path, AMLB task ID/name",
+    )
+    parser.add_argument("--metrics_dir", type=str, help="Directory to save metrics.")
+
+    args = parser.parse_args()
+    return args
+
+
+def _convert_torchvision_dataset(dataset):
+    from io import BytesIO
+
+    import pandas as pd
+
+    data = []
+    for i in range(len(dataset)):
+        x, y = dataset[i]
+        img_byte_arr = BytesIO()
+        x.save(img_byte_arr, format="PNG")
+        img_byte_arr = img_byte_arr.getvalue()
+        data.append((img_byte_arr, y))
+    df = pd.DataFrame(data, columns=["image", "label"])
+    return df
+
+
+def load_dataset(
+    data_path: str,  # can be dataset name or path to dataset
+):
+    if data_path in ["MNIST"]:
+        import torchvision.datasets as data
+
+        train_data = data.MNIST("./data", train=True, download=True)
+        test_data = data.MNIST("./data", train=False, download=True)
+        train_data = _convert_torchvision_dataset(train_data)
+        test_data = _convert_torchvision_dataset(test_data).sample(frac=0.1)
+        return train_data, test_data
+    else:
+        df = load_pd.load(data_path)
+        train_data, test_data = train_test_split(df, test_size=0.2, random_state=42)
```

Comment:
I believe this should have a `return` statement

Reply:
You are right.